recipes/qt/6.x.x/test_package/conanfile.py

```diff
@@ -36,8 +36,18 @@ def build(self):
         cmake.configure()
         cmake.build()
 
+    def _get_packageoption_value(self, optionname):
+        result = None
+        for (name, value) in self.options.values.as_list():
+            if name == f"qt:{optionname}":
+                result = value
+                break
+        return result
```

Comment:
```suggestion
        self.dependencies[self.tested_reference_str].options.get_safe(optionname)
```

Reply:
changed by untested commit 9dc56fa

Reply:
tested the changes on CI and locally for conan-v1. local builds with QT-declarative enabled also tested